src/components/player/index.jsx

```
@@ -0,0 +1,180 @@
+          )}
+        </div>
+      </Modal>
+
+      <Modal
+        title="Comments"
+        isOpen={isModalOpen}
+        onClose={() => setIsModalOpen(false)}
+      >
+        <div>
+          {comments?.length > 0 ? (
+            comments?.map((comment) => <div> {comment?.comment} </div>)
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)